test/reuse/util/file/getExcelData.spec.js

```diff
@@ -0,0 +1,45 @@
+const path = require("path");
+
+describe("file - getExcelData - Retrieve xls file data", function () {
```

Comment:
lets add some more tests, testing the different output formats

Reply:
@DIFSRIP I have added other output formats in test